cumulus/parachains/runtimes/assets/asset-hub-westend/src/lib.rs

```diff
@@ -1770,3 +1772,46 @@ cumulus_pallet_parachain_system::register_validate_block! {
 	Runtime = Runtime,
 	BlockExecutor = cumulus_pallet_aura_ext::BlockExecutor::<Runtime, Executive>,
 }
+
+parameter_types! {
+	// The deposit configuration for the singed migration. Specially if you want to allow any signed account to do the migration (see `SignedFilter`, these deposits should be high)
+	pub const MigrationSignedDepositPerItem: Balance = CENTS;
+	pub const MigrationSignedDepositBase: Balance = 2_000 * CENTS;
+	pub const MigrationMaxKeyLen: u32 = 512;
+}
+
+impl pallet_state_trie_migration::Config for Runtime {
+	type RuntimeEvent = RuntimeEvent;
+	type Currency = Balances;
+	type RuntimeHoldReason = RuntimeHoldReason;
+	type SignedDepositPerItem = MigrationSignedDepositPerItem;
+	type SignedDepositBase = MigrationSignedDepositBase;
+	// An origin that can control the whole pallet: should be Root, or a part of your council.
+	type ControlOrigin = frame_system::EnsureSignedBy<RootMigController, AccountId>;
+	// specific account for the migration, can trigger the signed migrations.
+	type SignedFilter = frame_system::EnsureSignedBy<MigController, AccountId>;
```

Comment:
For Polkadot and Kusama parachains we probably want this to be permissionless, or?  
Then we could also test it here?

Reply:
I think the idea is to use specific account eg bridge one (I would still prefer https://github.com/polkadot-fellows/runtimes/issues/74#issuecomment-1943400824 but this is more practical probably).
Maybe ControlOrigin is not strictly necessary, but SignedFilter is, otherwise we could have frontrunning issue (two ext on same block, then only one get refund, also we don't want multiple ext).